fe/pdb_writer.py

```diff
@@ -38,6 +38,34 @@
 
 from rdkit import Chem
 
+
+def convert_single_topology_mols(coords: np.ndarray, topo: "SingleTopology") -> np.ndarray:
```

Comment:
Question about type-hint syntax: Should this be `SingleTopology` or `"SingleTopology"`?

Reply:
Just didn't want to do the import, can add in the import. Not sure what impact the string style has on mypy

Reply:
Yeah this extra import dependency drove me crazy